.github/workflows/build-build-tools-image.yml

```diff
@@ -78,14 +79,17 @@ jobs:
           provenance: false
           push: true
           pull: true
-          file: Dockerfile.build-tools
-          cache-from: type=registry,ref=cache.neon.build/build-tools:cache-${{ matrix.arch }}
-          cache-to: ${{ github.ref_name == 'main' && format('type=registry,ref=cache.neon.build/build-tools:cache-{0},mode=max', matrix.arch) || '' }}
-          tags: neondatabase/build-tools:${{ inputs.image-tag }}-${{ matrix.arch }}
+          file: docker/debian/${{ matrix.os }}/Dockerfile.build-tools
+          cache-from: type=registry,ref=cache.neon.build/build-tools:cache-${{ matrix.arch }}-${{ matrix.os }}
+          cache-to: ${{ github.ref_name == 'main' && format('type=registry,ref=cache.neon.build/build-tools:cache-{0}-{1},mode=max', matrix.arch, matrix.os) || '' }}
+          tags: neondatabase/build-tools:${{ inputs.image-tag }}-${{ matrix.arch }}-${{ matrix.os }}
 
   merge-images:
     needs: [ build-image ]
     runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        os: [ bullseye, bookworm ]
```

Comment:
GitHub already has `runner.os`, using `matrix.os` is confusing

Reply:
runner and matrix both are different thing, we can use `code-name` which they call in Debian term or we can call it `debians`

Reply:
will update this with `debian_version`